releasenotes/notes/0.46/prepare-0.46.0-c3857463a3c052f7.yaml

```
@@ -0,0 +1,620 @@
+        :class:`.BasicAerJob`            |  :class:`.BasicProviderJob`
+        :class:`.QasmSimulatorPy`        |  :class:`.BasicSimulator`
+        :class:`.UnitarySimulatorPy`     |  use :class:`~.quantum_info.Operator`
+        :class:`.StatevectorSimulatorPy` |  use :class:`~.quantum_info.Statevector`
+
+
+    A notable difference is that the new provider is no longer exposed through a global instance
+    (like ``BasicAer``), so it will not be valid to do ``from qiskit import BasicProvider``.
+    Instead, the provider class must be imported from its submodule and instantiated::
+
+        from qiskit.providers.basic_provider import BasicProvider
+        provider = BasicProvider()
+        backend = provider.get_backend("sim_name")
+
+    The following examples show the migration paths of the three simulators in :mod:`.basicaer`.
+
+    1. Statevector simulator::
+
+        from qiskit import QuantumCircuit
+        qc = QuantumCircuit(3)
+        qc.h(0)
+        qc.h(1)
+        qc.cx(1,2)
+        qc.measure_all()
+
+        # Former path
+        from qiskit import BasicAer
+        backend = BasicAer.get_backend("statevector_simulator")
+        statevector = backend.run(qc).result().get_statevector()
+
+        # New path
+        qc.remove_final_measurements() # no measurements allowed
+        from qiskit.quantum_info import Statevector
+        statevector = Statevector(qc)
+
+    2. Unitary simulator::
+
+        from qiskit import QuantumCircuit
+        qc = QuantumCircuit(3)
+        qc.h(0)
+        qc.h(1)
+        qc.cx(1,2)
+        qc.measure_all()
+
+        # Former path
+        from qiskit import BasicAer
+        backend = BasicAer.get_backend("unitary_simulator")
+        result = backend.run(qc).result()
+
+        # New path
+        qc.remove_final_measurements() # no measurements allowed
+        from qiskit.quantum_info import Operator
+        result = Operator(qc).data
+
+    3. Qasm simulator::
+
+        from qiskit import QuantumCircuit
+        qc = QuantumCircuit(3)
+        qc.h(0)
+        qc.h(1)
+        qc.cx(1,2)
+        qc.measure_all()
+
+        # Former path
+        from qiskit import BasicAer
+        backend = BasicAer.get_backend("qasm_simulator")
+        result = backend.run(qc).result()
+
+        # New path
+        from qiskit.providers.basic_provider import BasicProvider
+        backend = BasicProvider().get_backend("basic_simulator")
+        result = backend.run(qc).result()
+        # or, directly
+        from qiskit.providers.basic_provider import BasicSimulator
+        backend = BasicSimulator()
+        result = backend.run(qc).result()
+  - |
+    Deprecated using a :class:`~.PauliList` as an observable that is implicitly
+    converted to a :class:`~.SparsePauliOp` with coefficients 1 when calling
+    :meth:`.Estimator.run`. Instead you should explicitly convert the argument
+    using ``SparsePauliOp(pauli_list)`` first.
```

I'm not sure if the space before the `-|` is important, but adding it just in case:

```suggestion
  
  - |
    Using a :class:`~.PauliList` as an observable that is implicitly
    converted to a :class:`~.SparsePauliOp` with coefficients 1 when calling
    :meth:`.Estimator.run` is deprecated. Instead you should explicitly convert the argument
    using ``SparsePauliOp(pauli_list)`` first.
```